packages/lisk-cryptography/README.md

```diff
@@ -1 +1,68 @@
-# lisk-cryptography
+# @liskhq/lisk-cryptography
+
+@liskhq/lisk-cryptography is containing general cryptographic functions for use with Lisk-related software
+
+## Installation
+
+```sh
+$ npm install --save @liskhq/lisk-cryptography
+```
+
+## Benchmarking
+
+Install optional dependencies:
+
+```sh
+$ npm i -D benchmark sodium-native
+```
+
+Benchmark nacl functions:
+
+```sh
+$ npx babel-node ./benchmark/nacl
+```
+
+Benchmark results for nacl functions:
+
+| Function            | Results                                                         | Winner |      
```

Comment:
Can you update this table in this format? it will be more redable

Function    |    Fast                                            | Slow                                           |          Winner      
-----------|---------------------------------|------------------------------|--------------      
box             |   x 23,982 ops/sec ±0.59%         |  x 771 ops/sec ±0.44%            |            Fast